integration-tests/load/ccip/ccip_test.go

```diff
@@ -60,32 +55,46 @@ func TestCCIPLoad_RPS(t *testing.T) {
 	env, err := crib.NewDeployEnvironmentFromCribOutput(lggr, cribDeployOutput)
 	require.NoError(t, err)
 	require.NotNil(t, env)
+	userOverrides.Validate(t, env)
 
 	// initialize loki using endpoint from user defined env vars
-	loki, err := wasp.NewLokiClient(wasp.NewLokiConfig(userOverrides.LokiEndpoint, nil, nil, nil))
+	loki, err := wasp.NewLokiClient(wasp.NewEnvLokiConfig())
 	require.NoError(t, err)
 	defer loki.StopNow()
 
+	// initialize additional accounts on other chains
+	transmitKeys, err := fundAdditionalKeys(lggr, &wg, *env, env.AllChainSelectors()[:*userOverrides.NumDestinationChains])
```

Comment:
Flagging the pointer dereference here, fine as long as we know that these pointers will be non-nil